UI/src/UI.cpp

```diff
@@ -4,9 +4,11 @@
 #include <glad/glad.h>
 #include <GLFW/glfw3.h>
 
-UI::UI() {
+UI::UI(Scene* s) {
```

Comment:
Same comment about single letter parameters.

Reply:
I changed that but looks like that file didn't save before I committed, will be in next commit